app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt

```diff
@@ -98,6 +118,16 @@ class SelectionInteractionViewModel private constructor(
     writtenTranslationContext = translationContext
   }.build()
 
+  override fun getRawUserAnswer(): RawUserAnswer = RawUserAnswer.newBuilder().apply {
+    itemSelection = ItemSelectionRawAnswer.newBuilder().apply {
+      if (interactionId == "ItemSelectionInput") {
+        addAllSelectedIndexes(selectedItems)
```

Comment:
Could we just use this? It doesn't seem like the if-check is actually needed anymore.